packages/stylelint-plugin/src/rules/rename-tokens.ts

```diff
@@ -0,0 +1,98 @@
+import { AtRule, Declaration, Root } from "postcss"
+import postcssValueParser from "postcss-value-parser"
+import { usageOfDeprecatedName } from "../messages"
+import { Options, RuleDefinition } from "../types"
+import { parseVariable } from "../util/variableUtils"
+import {
+  walkAtRulesWithKaizenTokens,
+  walkDeclsWithKaizenTokens,
+} from "../util/walkers"
+
+const renameRules: Array<[RegExp, string]> = [
+  [/color-wisteria/, "color-purple"],
+  [/color-cluny/, "color-blue"],
+  [/color-yuzu/, "color-yellow"],
+  [/color-coral/, "color-red"],
+  [/color-seedling/, "color-green"],
+  [/color-peach/, "color-orange"],
+  [/color-white/, "color-neutral-100"],
+  [/color-stone/, "color-neutral-200"],
+  [/color-ash/, "color-neutral-300"],
+  [/color-iron/, "color-neutral-500"],
+  [/color-slate/, "color-neutral-600"],
+  [/kz-var-/, ""],
```

Comment:
What are we going to do about the legacy `kz-` tokens (eg `kz-color-coral-400`). 

These were hardcoded values (hex for colours, etc). Are these going to exist in some form? (eg `color-coral-400-hex`)

And if they do, are we going to handle the replacements here?

Reply:
This comment probably needs to go on #1741